Examples/Run/Common/include/ActsExamples/Options/CommonOptions.hpp

```diff
@@ -41,7 +41,18 @@ void addMaterialOptions(boost::program_options::options_description& opt);
 void addInputOptions(boost::program_options::options_description& opt);
 
 /// Add common output-related options.
-void addOutputOptions(boost::program_options::options_description& opt);
+enum OutputFormat {
+  None = 0,
+  Root = 1,
+  Csv = 2,
+  Obj = 4,
+  Json = 8,
+  Txt = 16,
+  All = Root | Csv | Obj | Json | Txt
```

Comment:
Alternatively: `std::numeric_limit<int>::max()`, which should be all 1s, which remains valid if new options were added.

Reply:
Good idea, indeed.